tests/test_main.py

```diff
@@ -1 +1,88 @@
-# write your code here
+from app.main import get_human_age
+
+
+def test_output_type():
+    assert type(get_human_age(1, 1)) == list
+
+    is_int = True
+    if type(get_human_age(1, 1)) == list:
```

Comment:
You don't need that condition, you assert that it is a list above.

Reply:
Used pytest.mark.parametrize